Dockerfile-installer.arm32v7

```diff
@@ -0,0 +1,48 @@
+# Specify the version of PHP we use for our Chevereto
+ARG PHP_VERSION=7.3-apache
+FROM alpine as downloader
+
+RUN apk add --no-cache curl && \
+    mkdir -p /extracted/Chevereto/images && \
+    cd /extracted/Chevereto && \
+    curl -sS -o index.php -L "https://chevereto.com/download/file/installer" && \
+    curl -L https://github.com/balena-io/qemu/releases/download/v4.0.0%2Bbalena2/qemu-4.0.0.balena2-arm.tar.gz | tar zxvf - -C .  && \
+    mv qemu-4.0.0+balena2-arm/qemu-arm-static .
+COPY settings.php /extracted/Chevereto/app/settings.php
+
+FROM arm32v7/php:$PHP_VERSION
```

Comment:
My guess is that we have to use this to force the underlying `php` image to be of correct architecture. Is that the case?